kafka-bridge/src/main/java/dk/dbc/holdingsitems/kafkabridge/JobProcessor.java

```diff
@@ -53,6 +54,7 @@
     EntityManager em;
 
     @Timed
+    @TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
```

Comment:
Hvad betyder den her? Hvorfor er der brug for den?

Reply:
Det betyder at den EntityManager der er i klassen skal have en transaktion som slutter når metoden returnerer ikke når tråden terminerer. Da der ikke længere bruges den connection, som kommer med fra pg-queue.